src/content/controller_content.ts

```diff
@@ -184,13 +207,17 @@ function* ParserController() {
                 )
 
                 if (response.isMatch && keyevent instanceof KeyboardEvent) {
-                    keyevent.preventDefault()
-                    keyevent.stopImmediatePropagation()
                     canceller.push(keyevent)
                 }
 
                 if (response.exstr) {
                     exstr = response.exstr
+                    if (exstr.startsWith("fillcmdline")) { // TODO That's ugly. I need a way to know if this command is going to open the command line. Is there a better way?
```

Comment:
We need to exclude _nofocus and _tmp because they're used for displaying messages and don't get focus.

I don't think there's a better way to know. We probably need to have some sort of function decorator...

Reply:
Indeed, so it's down to `fillcmdline` and `fillcmdline_notrail` only?

Reply:
Yeah, exactly. Otherwise this looks good. I'll probably release it in a beta in early January